diff -Nuar pinktrace-0.1.2.orig/configure pinktrace-0.1.2/configure
--- pinktrace-0.1.2.orig/configure	2011-06-28 16:55:18.000000000 +0300
+++ pinktrace-0.1.2/configure	2015-10-14 18:34:15.668384255 +0300
@@ -12079,7 +12079,7 @@
 $as_echo_n "checking for supported kernel... " >&6; }
 	kernel_version=`uname -r | cut -d'-' -f1`
 	case "$kernel_version" in
-	3.*|2.6*)
+	4.*|2.6*)
 		{ $as_echo "$as_me:${as_lineno-$LINENO}: result: $kernel_version" >&5
 $as_echo "$kernel_version" >&6; }
 		;;
diff -Nuar pinktrace-0.1.2.orig/configure.ac pinktrace-0.1.2/configure.ac
--- pinktrace-0.1.2.orig/configure.ac	2011-06-28 16:49:35.000000000 +0300
+++ pinktrace-0.1.2/configure.ac	2015-10-14 18:34:39.990928585 +0300
@@ -104,7 +104,7 @@
 	AC_MSG_CHECKING([for supported kernel])
 	kernel_version=`uname -r | cut -d'-' -f1`
 	case "$kernel_version" in
-	3.*|2.6*)
+	4.*|2.6*)
 		AC_MSG_RESULT([$kernel_version])
 		;;
 	2.5*)
