--- colors/kib_plastic.vim	2007-07-07 12:15:11.000000000 +0200
+++ colors/kib_plastic.vim	2007-07-07 12:46:10.000000000 +0200
@@ -172,7 +172,7 @@
 hi Visual	    guibg=#ff6a6a guifg=#000000             " Selection of text in Visual Mode
 "hi VisualNOS
 hi WarningMsg	guifg=salmon
-hi WildMenu
+"hi WildMenu
 hi Menu        guibg=#ff6a6a guifg=#000000
 "hi Scrollbar   guibg=grey30 guifg=tan
 "hi Tooltip
