--- colors/potts.vim	2004-09-01 11:31:16.000000000 +0200
+++ colors/potts.vim	2007-07-07 12:48:35.000000000 +0200
@@ -37,7 +37,7 @@
 highlight DiffText term=reverse cterm=bold ctermfg=15 ctermbg=12 gui=bold guifg=White guibg=Red
 highlight Cursor guifg=bg guibg=fg
 highlight lCursor guifg=bg guibg=fg
-highlight Normal guifg=gray guibg=black font='6x13'
+highlight Normal guifg=gray guibg=black
 highlight Comment term=bold ctermfg=11 guifg=#80a0ff
 highlight Constant term=underline ctermfg=13 guifg=#ffa0a0
 highlight Special term=bold ctermfg=12 guifg=Orange
